### DISCUSSION TITLE
機器學習進展 降低語音複製門檻

### DISCUSSION THREAD

**Discussion by HSU-YU-JEN** [1101-GS4719-C/Python-Community-GS4719-C#64] [科技新知]
現在機器學習與類神經網路，已經被用於複製人聲，也就是說~電腦可以從你所錄製的一段話中學習你的聲音，並將它轉化合成為任何一段話

網址文章:https://www.inside.com.tw/article/24850-ai-voice-clone-synthesis-deepfake-applications-vergecast

**Comment by enoch-lightning** [comment]
深偽技術(Deepfake)為使用深度學習於照片或影片上，更改原有人物或聲音，以假亂真。我認同其確實為雙面刃，既能為生活帶來便利，也能造成社會的混亂。

**Reply by xCk27x** [reply]
[自 2017 年，某 Reddit 用戶首次用來偽造色情影片開始，Deepfake 造假逐漸滲透到政治、媒體等多領域，甚至還威脅到今年美國總統大選](https://technews.tw/2020/11/28/deepfake-2020-development-status/)

過去Deepfake(深偽)技術多以影像結合與面容替換為主，已有許多惡意的深偽影片被製作，用來進行詆毀或攻擊，對影視界及政治界造成許多不良影響。如今連聲音都可以直接利用AI進行偽造，意味將影像與聲音的兩種深偽技術結合後，將能製作出更容易使人相信的假影片，對社會安全的危害可想而知。
我認為這項聲音模擬的新技術所造成的負面效果會遠大於正面效果，應當盡早研發出辨識聲音真假的技術。
